portal_pages/forms.py

```diff
@@ -103,6 +103,12 @@ class CaseStudyForm(SpamProtectedForm, forms.ModelForm):
 
     required_css_class = 'required'
 
+    marketplace_entry = forms.ModelChoiceField(queryset=None, required=False)
```

Comment:
I don't believe this is necessary? This field is already part of the form by virtue of this being a ModelForm for a case study page. We're going to override the value for queryset in init anyway, and it will have been made non-required by the model form anyway based on the model field value being blank=True...